scale/ingest/triggers/ingest_trigger_condition.py

```diff
@@ -54,4 +60,14 @@ def is_condition_met(self, source_file):
         if self._media_type and self._media_type != source_file.media_type:
             return False
 
-        return self._data_types <= source_file.get_data_type_tags()
+        condition_met = True
+        file_data_types = source_file.get_data_type_tags()
+
+        if self._not_data_types:
+            condition_met = True not in [tag in file_data_types for tag in self._not_data_types]
+        if self._any_data_types:
+            condition_met = True in [tag in file_data_types for tag in self._any_data_types]
+        if self._data_types:
+            condition_met = self._data_types <= file_data_types
```

Comment:
Does this logic work? The conditions don't seem to affect each other and it seems like the last condition checked is the only one that matters: `condition_met = self._data_types <= file_data_types`
Do your unit tests test the case where all the fields are provided and either not_data_types or any_data_types fail?

Also update `get_triggered_message` to include the two new fields.